src/modules/requests/index.ts

```diff
@@ -12,7 +12,7 @@ export * from "./incoming/received/ReceivedIncomingRequestParameters"
 export * from "./incoming/requireManualDecision/RequireManualDecisionOfIncomingRequestParameters"
 export * from "./itemProcessors/AbstractRequestItemProcessor"
 export * from "./itemProcessors/createAttribute/CreateAttributeRequestItemProcessor"
-export * from "./itemProcessors/FreeTextAttribute/FreeTextRequestItemProcessor"
+export * from "./itemProcessors/freeTextAttribute/FreeTextRequestItemProcessor"
```

Comment:
```suggestion
export * from "./itemProcessors/freeText/FreeTextRequestItemProcessor"
```
that item isn't related to an attribute -> createAttribute is for the CreateAttributeRequestItem